contracts/InterestRate/StableRate.sol

```diff
@@ -0,0 +1,248 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.13;
+
+/**
+ * @title Logic for Venus stable rate.
+ */
+contract StableRateModel {
+    event NewStableInterestParams(
+        uint256 baseRatePerBlockForStable,
+        uint256 multiplierPerBlock,
+        uint256 multiplierPerBlockForStable,
+        uint256 jumpMultiplierPerBlockForStable,
+        uint256 kink,
+        uint256 stableRatePremium,
+        uint256 optimalStableLoanRate
+    );
+
+    /// @notice Indicator that this is an InterestRateModel contract (for inspection)
+    bool public constant isInterestRateModel = true;
+
+    uint256 private constant BASE = 1e18;
+
+    /**
+     * @notice The approximate number of blocks per year that is assumed by the interest rate model
+     */
+    uint256 public constant blocksPerYear = 2102400;
+
+    /**
+     * @notice The stable base interest rate which is the y-intercept when utilization rate is 0
+     */
+    uint256 public baseRatePerBlockForStable;
+
+    /**
+     * @notice The multiplier of utilization rate that gives the slope of the interest rate
+     */
+    uint256 public multiplierPerBlock;
+
+    /**
+     * @notice The multiplier of utilization rate that gives the slope of the stable interest rate
+     */
+    uint256 public multiplierPerBlockForStable;
+
+    /**
+     * @notice The multiplierPerBlock after hitting a specified utilization point for stable rate
+     */
+    uint256 public jumpMultiplierPerBlockForStable;
+
+    /**
+     * @notice The utilization point at which the jump multiplier is applied for stable rate
+     */
+    uint256 public kink;
+
+    /**
+     * @notice The premium rate applicable on optimal stable loan rate
+     */
```

Comment:
```suggestion
     /// @notice The premium rate applied on top of the optimal stable loan rate
```

Reply:
Fixed, can check the changes at [this commit](2981e0d08b247b7c7a7dc3461e9701d7d593c520).